cloudera/datadog_checks/cloudera/check.py

```diff
@@ -15,29 +15,30 @@ class ClouderaCheck(AgentCheck, ConfigMixin):
 
     def __init__(self, name, init_config, instances):
         super(ClouderaCheck, self).__init__(name, init_config, instances)
+        self.log.debug('name:%s init_config:%s, instances:%s', name, init_config, instances)
```

Comment:
Nit: I think this should be removed before merging, since configuration parsing wouldn't be the responsibility of the cloudera check but rather the base check.